spinoso-time/src/time/tzrs/parts.rs

```diff
@@ -216,7 +218,7 @@ impl Time {
     #[inline]
     #[must_use]
     pub fn is_utc(&self) -> bool {
-        Offset::Utc == self.offset
+        OffsetType::Utc == self.offset.inner
```

Comment:
nit, but for matching a single variant like this to implement a boolean accessor, this codebase tends to prefer `matches!` because it can be `const`:

```suggestion
        matches!(self.offset.inner, OffsetType::Utc)
```

like:

https://github.com/artichoke/artichoke/blob/c45f0abcb8799f328db8e7a603f5da490443dc9a/spinoso-string/src/codepoints.rs#L102-L112